### PR TITLE
Clean up uFcontext build process

### PR DESCRIPTION
**Pull request by evan-charmworks**
Fixes `-win64` builds building i386 assembly files

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Is this a followup from #2728?

**Comment by evan-charmworks** [comment]
> Is this a followup from #2728?

Yes

File: src/util/boost-context/import.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```
Comment (matthiasdiener) [inline src/util/boost-context/import.sh]: Can you add a comment here on when this file should be run? (I'm guessing when updating boost)

Comment (evan-charmworks) [reply src/util/boost-context/import.sh]: Done

**Comment by evan-charmworks** [comment]
> There was a UCX build failure, but that was spurious?

It was the same issue that 32-bit `netlrts-linux` autobuilds are having, which is not related to this PR.

**Comment by evan-charmworks** [comment]
Needs one more review

**Pull request merged**